<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaceForDecryption</class>
 <widget class="QWidget" name="interfaceForDecryption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sendBackToinit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>return to init window</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="enterHash">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="hashToText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>471</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="decrypt_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>decrypt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyForDecryption">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
